block/manager.go

```diff
@@ -295,6 +295,21 @@ func (m *Manager) GetStoreHeight() uint64 {
 	return m.store.Height()
 }
 
+// GetDAIncludedHeight returns the manager's da included height
+// FIXME(tuxcanfly): this is in accurate before first retrieve loop
+func (m *Manager) GetDAIncludedHeight(ctx context.Context) uint64 {
```

Comment:
This just gets the last DA included height. 
Don't we want the DA included height by block? To know what height a block was included at? 

How do you map the rollup block height to the da included height?